dev-docs/bidders/deltaprojects.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: bidder
+title: Delta Projects
+description: Prebid Delta Projects Bidder Adapter
+biddercode: deltaprojects
+gdpr_supported: true
+gvl_id: 209
+media_types: banner
+pbjs: true
+---
+
+### Note
+Contact publishers@deltaprojects.com to get a publisher id and to agree on a currency. Delta Projects
+will always bid and log values in the agreed upon currency, utilizing the currency module if necessary and available.
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name           | Scope    | Description                                                              | Example        | Type     |
+|----------------|----------|--------------------------------------------------------------------------|----------------|----------|
+| `publisherId`  | required | Publisher ID from Delta Projects. Contact publishers@deltaprojects.com   | `'4'`          | `string` |
```

Comment:
It's always helpful for testing to have

- a full example of the bid object as
- a test placement that a publisher can use to check if the integration works